test/e2e/app-dir/segment-cache/segment-prefetch-config/app/prefetch-static/page.tsx

```diff
@@ -0,0 +1,20 @@
+import { connection } from 'next/server'
+import { Suspense } from 'react'
+
+export const prefetch = 'static'
```

Comment:
Since this is extremely likely to change, let's prefix this and make it look slightly more like what we expect the final API to be:

Suggestions:

Static

```js
export const unstable_prefetch = {}
```

Dynamic

```js
export const prefetch = {
  unstable_forceRuntime: true

  // or
  unstable_runtimeSamples: anyNonEmptyArray
}
```